fr/community/index.md

```diff
@@ -59,6 +59,6 @@ Informations générales
 
 
 [ruby-central]: http://rubycentral.org/
-[ruby-discord]: https://ruby-discord.com/
```

Comment:
This link was added in #2762 by an admin of Discord Ruby, and it redirects to `https://discord.gg/ruby-518658712081268738` properly. Will `https://ruby-discord.com/` be more stable?

Reply:
Technically the ruby-discord.com link is okay. But it's not necessarily the kind of link one expects to see when getting a discord invite since it's not from discord.gg. So we want to use an actual discord domain link to avoid any suspicion of a scam.

Great point about stability though. I was kinda panicked and trying to restore the link as quick as possible and didn't think about it. The link I gave is actually our vanity url which isn't going to be stable, and actually as of the time of this writing, I just disabled due to another discord bug.

I will edit again momentarily with a never-expiring link to our recently created #getting-started channel. It _should_ remain stable unless Discord loses this invite like our old one.

FWIW I am one of the discord admins :)

Reply:
Thanks for the clarification. I'm okay with the `discord.gg/` link.